lib/content-services/src/lib/dialogs/download-zip/download-zip.dialog.stories.component.ts

```diff
@@ -1,61 +0,0 @@
-/*!
- * @license
- * Copyright © 2005-2025 Hyland Software, Inc. and its affiliates. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { Component, Input, OnInit, OnChanges } from '@angular/core';
-import { MatDialog } from '@angular/material/dialog';
-import { DownloadZipDialogComponent } from './download-zip.dialog';
-import { zipNode, downloadEntry } from './mock/download-zip-data.mock';
```

Comment:
Hmm I think we can remove zipNode and downloadEntry completely too now. I know it shows usages of them in download-zip-service.mock.ts file but to be honest I can't see usages of that service mock file too anywhere so that can be removed as well.